assets/js/atomic/blocks/product-elements/button/frontend.tsx

```diff
@@ -86,20 +124,108 @@ const productButtonSelectors = {
 			state,
 		}: {
 			context: Context;
-			// For now, let's just use any.
-			// eslint-disable-next-line @typescript-eslint/no-explicit-any
 			selectors: any;
 			state: State;
 		} ) => {
-			return selectors.woocommerce.isThereMoreThanOneItem( {
-				context,
-				state,
-			} );
+			return (
+				selectors.woocommerce.isThereMoreThanOneItem( {
+					context,
+					state,
+				} ) && context.woocommerce.displayViewCart
+			);
+		},
+		hasCartLoaded: ( { state }: { state: State } ) => {
+			return state.woocommerce.cart !== undefined;
+		},
+		numberOfItems: ( {
```

Comment:
Given that this is a function, I think it would be great to start its name with a verb, something like `getNumberofItems()`. What do you think?